cmds/core/timeout/main_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"errors"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+// TestOK for now just runs a simple successful test with 0 args or more than one arg.
+func TestOK(t *testing.T) {
```

Comment:
this test tests errors... so maybe a name other than TestOK makes sense here?

Reply:
Yep. It's now TestBadInvocation